project/npda/templates/partials/patient_table.html

```diff
@@ -4,8 +4,92 @@ <h1 class="text-md font-montserrat font-semibold text-rcpch_dark_blue">Patients
 <table class="table table-md w-full text-sm text-left rtl:text-right text-gray-500 text-gray-400 mb-5 font-montserrat">
     <thead class="text-xs text-gray-700 uppercase bg-gray-50 bg-rcpch_dark_blue text-white">
             <tr>
-                <th scope="col" class="px-2 py-3 text-center">NPDA ID</th>
-                <th scope="col" class="px-2 py-3 text-center">NHS Number</th>
+                <th scope="col" class="px-2 py-3 text-center cursor-pointer">
+                        <span class="flex flex-row">
+                            <?xml version="1.0" ?>
```

Comment:
I wonder if we make a partial for each SVG icon just for readability?

Reply:
this is a great suggestion - I have done this.